app/Http/Controllers/SubscribeController.php

```diff
@@ -147,15 +148,19 @@ public function showManage($code = null)
         }
 
         $subscriber = Subscriber::where('verify_code', '=', $code)->first();
+        $usedComponentGroups = Component::enabled()->where('group_id', '>', 0)->groupBy('group_id')->pluck('group_id');
```

Comment:
@CachetHQ/owners I feel like we need a better way to handle this and the next 2 lines too.